build/IPDK_Container/docker/README_DOCKER.md

```diff
@@ -0,0 +1,223 @@
+$ systemctl restart docker
+```
+
+Note: Build and run your IPDK Fedora33 container from `build/IPDK_Container`
+directory or Ubuntu20.04 container from `build/IPDK_Container/Ubuntu20.04`
+directory in `https://github.com/ipdk-io/ipdk` repository.
+
+### Source Code availability
+By default source code for P4-OVS, p4-driver and P4C will not be retained.
+If user wants to retain source code, a variable (`KEEP_SOURCE_CODE`) in
+docker.env file is available. User can modify this variable to get results
+as mentioned below.
+
+1) If `KEEP_SOURCE_CODE=NO` which is a default value then, complete source code
+for P4-OVS, p4-driver and P4C will be removed.
+
+2) `KEEP_SOURCE_CODE=YES` then, source code for P4-OVS, p4-driver and P4C will
+be retained but temporary files generated after building these repositories
+will be removed.
+
+### Commands to bring-up container and run OvS with P4
+* If you are behind a proxy, edit proxy parameter in `docker.env` and 
+`IPDK_Container/start_p4ovs.sh` files to add your proxy.
+   
+* The following commands can be used to build:
+    1. Run `./ipdk build --no-cachec` - To build a docker image while no proxy.
+    2. Run `./ipdk build --no-cache --proxy ` - To build a docker image while
+       running behind proxy.
+    3. Run `./ipdk build` - To build a docker image while no proxy, with cached
+       previous build's data.
+    4. Run `./ipdk build --proxy` - To build a docker image while running
+       behind proxy, with cached previous build's data.
+
+* Now the container image is build. The P4OVS switch can be started as docker
+container daemon with `./ipdk start -d`. A `volume` directory will be created
+in the current working directory. This directory will be used for creating the
+vhost socket interfaces in `volume/intf`, all the log files in `volume/logs`
+and can be used to share files with the daemon container where it is available
+as '/tmp'.
+
+* Run - './ipdk connect' - To connect to your IPDK container daemon and to use
+it from a command line.
+
+If above commands are successful, at this point you should have your IPDK 
+container up and running. You should see the container prompt like below:
+
+```
+    root@c5efb1a949ad ~]#
+```
+
+`c5efb1a949ad` is your container id as displayed in command `docker ps -a`
+
+As your P4-OvS should be up and running, you should see the ovsdb-server
+and ovs-vswitchd process with the ‘ps -ef | grep ovs’ command.
+
+## Section 2: Example scenario setup ( VM1 <-> IPDK Container <-> VM2 )
+Below commands will help you setup traffic between 2 VMs on host with 
+IPDK container as a vswitch switching traffic between them:
+
+Pre-requisite: 
+- Container in Section 1 should be up and with OvS running
+
+### Commands to setup the example scenario:
+
+1) Startup a clean IPDK container daemon with './ipdk start -d'.
+
+2) Create 2 vhost ports using GNMI commands through the 'ipdk' script:
+    a. ./ipdk addintf --name net_vhost0 --host host1 --socket vhost-user-0 --type LINK
```

Comment:
Since this is a markdown file, it's nice to make example commands wrapped with backticks so users can easily copy and paste them into a terminal. I'd do this everywhere in this file where you have sample commands.